docs/README.rst

```diff
@@ -1046,6 +1046,71 @@ functionality for particular interfaces.
    how to override functions in interface definitions and why, prior
    to Python 3.6, the zero-argument version of `super` cannot be used.
 
+
+Persistency and Equality
+========================
+
+An important design goal has been persistency support for interfaces.
+This allows different processes to use the same interface and
+to create persistent associations between (persistent) objects
+and interfaces. For example, an application can store an object
+together with its provided interfaces in a database; later, potentially
+in a different invocation, it can search the database for objects
+providing a given interface.
+
+To make an object persistent, it must have a persistent identifier,
```

Comment:
To avoid getting into the weeds too much, I would probably gloss over some of the details of persistence and ``pickle`` in particular and just talk about "stable identifiers" and "global objects".